stable/insights-agent/templates/helm-hook/job.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  {{- include "metadata" "cronjob-executor" }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+spec:
+  ttlSecondsAfterFinished: 30
+  backoffLimit: {{ .Values.cronjobs.backoffLimit }}
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ include "insights-agent.fullname" . }}-cronjob-executor
+      containers:
+      - name: start-job-from-cronjob
+        image: bitnami/kubectl:{{ .Capabilities.KubeVersion.Major }}.{{ .Capabilities.KubeVersion.Minor }}
+        command: ["bash"]
+        args:
+          - -c
+          - |
+            kubectl get cj --selector=app=insights-agent -o wide | grep insights | cut -d' ' -f1 | xargs -I~job~ sh -c "kubectl delete job ~job~ || echo 'Does Not Exist'"
+            kubectl get cj --selector=app=insights-agent -o wide | grep insights | cut -d' ' -f1 | xargs -I~job~ kubectl create job ~job~ --from=CronJob/~job~
```

Comment:
OOC, why not do `-o name` and skip the `cut`? Also why `grep insights`? Guessing the two are related.

Reply:
Whoops :)